<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Liste des projets</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.png</normaloff>icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font: 10pt;
}
#Form{
	
	background-color: #eee;
	font: 10pt;
}

.QPushButton{
	border-radius: 2;
	background-color: qlineargradient(spread:pad, x1:0.548, y1:1, x2:0.469455, y2:0, stop:0.627119 rgba(242, 101, 101, 255), stop:1 rgba(255, 146, 117, 255));
	border-bottom: 2px solid;
	border-bottom-color:rgb(221, 83, 83);
	padding: 3px 16px;
	color:rgb(255, 221, 221);	
	font: 10pt;
}
.QPushButton:pressed{
	background-color: rgb(220, 91, 91);
	border-bottom-color: rgb(194, 73, 73);
}

.QLineEdit{
	border-radius: 2;
	border-bottom: 2px solid rgb(166, 166, 166);
	font: 10pt;
	padding: 3px 6px;
	
	background-color:rgb(222, 222, 222);
}

.QComboBox{
	border-radius: 2;
	background-color:rgb(222, 222, 222);
	border: none;
	padding: 3px 6px;
	border-bottom: 2px solid rgb(166, 166, 166);
}
.QComboBox::drop-down{
	 subcontrol-origin: padding;
    subcontrol-position: top right;
	
	border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid;
}
QComboBox:!editable:on, QComboBox::drop-down:editable:on {
    background: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
                                stop: 0 #D3D3D3, stop: 0.4 #D8D8D8,
                                stop: 0.5 #DDDDDD, stop: 1.0 #E1E1E1);
}
QTableWidget{
	
	background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="2">
    <widget class="QPushButton" name="rechercher_button">
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QTableWidget" name="tableWidget">
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QAbstractScrollArea::AdjustToContents</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionMode">
      <enum>QAbstractItemView::MultiSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="textElideMode">
      <enum>Qt::ElideNone</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>70</number>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>50</number>
     </attribute>
     <attribute name="verticalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item row="1" column="3">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="ajouter_button">
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="supprimer_button">
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="annuler">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="input">
     <property name="placeholderText">
      <string>Code Projet</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>annuler</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>546</x>
     <y>428</y>
    </hint>
    <hint type="destinationlabel">
     <x>503</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
